src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -12,13 +12,32 @@
     [status-im.constants :as constants]
     [status-im.contexts.wallet.common.utils :as utils]
     [status-im.contexts.wallet.create-account.style :as style]
-    [status-im.feature-flags :as ff]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]
     [utils.responsiveness :refer [iphone-11-Pro-20-pixel-from-width]]
     [utils.security.core :as security]
     [utils.string]))
 
+(defn prepare-new-keypair
```

Comment:
hmm, this file is starting to bloat out. this file should really just be the `view`, can you move this to a utils file?
ideally src/status_im/contexts/wallet/create_account/utils.cljs so the domain of that utils is narrow in scope 👍